packages/nextjs/src/client-boundary/uiComponents.tsx

```diff
@@ -37,6 +54,9 @@ export const SignIn = (props: SignInProps) => {
 export const SignUp = (props: SignUpProps) => {
```

Comment:
It seems that you're not using the HOC for the components that are mounted in paths we can infer using the env variables.

I feel like we should align this behavior by either: 
- making the Hoc generic enough to support both cases
- supported env variables for all components

Do you want to discuss offline for possible solutions? :)

Reply:
☝️

Reply:
Yep I thought about that too, but the `path` of `<SignIn/>` and `<SignUp/>` can be also defined from the Clerk options on the `<ClerkProvider/>` and I didn't know if we wanted to make the changes to have ENV and options for every component.